.github/actions/setup-argocd-image-updater/index.js

```diff
@@ -0,0 +1,123 @@
+const path = require('path');
```

Comment:
This file is a copy/paste correct?

Reply:
Yes, this is mostly a copy of the `setup-infra` action which is itself a copy of the `setup-terragrunt` action in `infrahq/infrastructure`